scripts/Package_BGA/bga.py

```diff
@@ -25,6 +25,11 @@ def generateFootprint(config, fp_params, fp_id):
     layoutX = fp_params["layout_x"]
     layoutY = fp_params["layout_y"]
     
+    if "additional_tags" in fp_params:
+        additionalTag = fp_params["additional_tags"]
```

Comment:
Put a space in front of this so there isn't a space added in the .format() statement for each conditions.